common/tests/unit/ProfileTest.php

```diff
@@ -0,0 +1,169 @@
+<?php namespace common\tests;
+
+use common\models\Profiles;
+
+class ProfileTest extends \Codeception\Test\Unit
+{
+    /**
+     * @var \common\tests\UnitTester
+     */
+    protected $tester;
+    
+    protected function _before()
+    {
+    }
+
+    protected function _after()
+    {
+    }
+
+    // tests
+    public function testUserId_isIntAndExists_True()
+    {
+        $profile = new Profiles();
+        //it must exist a user id 1 in profiles table;
+        $profile->userId = 1;
+        $this->assertTrue($profile->validate(["userId"]));
+
+    }
+
+    // tests
+    public function testUserId_isIntAndNotExists_False()
+    {
+        $profile = new Profiles();
+        //it must not exist a user id -1 in profiles table;
+        $profile->userId = -1;
+        $this->assertFalse($profile->validate(["userId"]));
+    }
+
+    // tests
+    public function testUserId_isString_False()
+    {
+        $profile = new Profiles();
+        //it must not exist a user id -1 in profiles table;
```

Comment:
remove duplicate comments in all functions as they are only relevant to the function above